src/python/dxpy/nextflow/nextflow_utils.py

```diff
@@ -62,50 +64,20 @@ def get_regional_options(region):
     return regional_options
 
 def get_nextflow_assets(region):
+    basepath = path.dirname(__file__)
```

Comment:
Shouldn't it be dxpy.__file__?

Reply:
Why dxpy.__file__? I was trying to point to the current file, which is __file__ regardless of where its location is, or is there something I am missing?

Reply:
I guess Ola is suggesting to use the same code for getting the template
https://github.com/dnanexus/dx-toolkit/blob/70a583e20a6eeb5a9d4e74e165877b00314c6804/src/python/dxpy/nextflow/nextflow_utils.py#L29

Reply:
Ah, right, fixed to stay consistent (it had the same functionality, just dxpy.__file__ points to a different file)